app/controllers/container_node_controller.rb

```diff
@@ -7,6 +7,10 @@ class ContainerNodeController < ApplicationController
   after_action :cleanup_action
   after_action :set_session_data
 
+  def show_list
+    process_show_list(:where_clause => 'container_nodes.deleted_on IS NULL')
```

Comment:
Scope exists now.  I asked on gitter how to use it, try `process_show_list(:named_scope => :active)`.